 <launch>
    <arg name="prefix" default="unity"/>
    <arg name="joystick" default="true"/>
    <arg name="keyboard" default="true"/>
    <arg name="rviz" default="true"/>

    <group ns="$(arg prefix)">
        <!-- Remap haptic topic -->
        <remap from="/unity/device1/pose" to="/haptic/device1/pose"/>
        <remap from="/unity/device1/joint_states" to="/haptic/device1/joint_states"/>
        <remap from="/unity/device1/button" to="/haptic/device1/button"/>

        <!-- Run joystick -->
        <!-- Set param interface, teleop_state -->
        <!-- Run move_group_python_interface -->
        <!-- Run mode_manager -->
        <include file="$(find ur10_teleop_interface)/launch/teleop_interface.launch">
            <arg name="prefix" value="$(arg prefix)"/>
            <arg name="joystick" value="$(arg joystick)"/>
            <arg name="keyboard" value="$(arg keyboard)"/>
            <arg name="rviz" value="$(arg rviz)"/>
        </include>
    </group>
 </launch>    